pydata_sphinx_theme/icon-links.html

```diff
@@ -1,20 +1,20 @@
 {%- macro icon_link_nav_item(url, icon, name) -%}
   {%- if url | length > 2 %}
         <li class="nav-item">
-          <a class="nav-link" href="{{ url }}" rel="noopener" target="_blank" title="{{ _(name) }}">
+          <a class="nav-link" href="{{ url }}" rel="noopener" target="_blank" title="{{ name }}">
             <span><i class="{{ icon }}"></i></span>
-            <label class="sr-only">{{ _(name) }}</label>
+            <label class="sr-only">{{ name }}</label>
           </a>
         </li>
   {%- endif -%}
 {%- endmacro -%}
 
-      <ul id="navbar-icon-links" class="navbar-nav" aria-label="{{ _(theme_icon_links_label) }}">
+      <ul id="navbar-icon-links" class="navbar-nav" aria-label="{{ theme_icon_links_label }}">
```

Comment:
It’s a bit unclear to me why some now are not translated with _ while others are. Could you provide a quick explanation in the contributing docs perhaps?

Reply:
You would need an oracle to know the translation of a variable string :) But I'll add a note.

Reply:
Done in 7ac36e4f